diff -u -r --from-file=gentoo-0.11.55/ gentoo-0.11.55/src/configure.c gentoo-0.11.55-home_etc/src/configure.c
--- gentoo-0.11.55/src/configure.c	2003-10-12 15:45:23.000000000 +0200
+++ gentoo-0.11.55-home_etc/src/configure.c	2005-06-24 00:11:12.000000000 +0200
@@ -173,14 +173,22 @@
 	guint		i;
 	const CfgModule	*page;
 
+	if((home = getenv("HOME_ETC")) != NULL)
+	{
+		strcpy(rcname, home);
+		strcat(rcname, "/" RCNAME);
+	}
+
+	else { 
 	if((home = getenv("HOME")) != NULL)
 	{
 		strcpy(rcname, home);
 		strcat(rcname, "/" RCNAME);
+
 	}
 	else
 		return;
-
+	}
 	cfg_modified_clear(min);
 	if((out = xml_put_open(rcname)) != NULL)
 	{
@@ -440,9 +448,15 @@
 	gchar	name[PATH_MAX] = "", *hpath;
 	guint32	i, flags = 0UL;
 
-	if((hpath = getenv("HOME")) != NULL)
+	if((hpath = getenv("HOME_ETC")) != NULL) {
 		g_snprintf(name, sizeof name, "%s" G_DIR_SEPARATOR_S "%s", hpath, RCNAME);
-
+	}
+	else {
+	if((hpath = getenv("HOME")) != NULL) {
+		g_snprintf(name, sizeof name, "%s" G_DIR_SEPARATOR_S "%s", hpath, RCNAME);
+		}
+	}
+	
 	/* Does the user seem to have a local config? */
 	if(!fut_can_read_named(name))
 		g_snprintf(name, sizeof name, PATH_CFG G_DIR_SEPARATOR_S "%s", RCNAME + 1);	/* Nope, check for global one. */
@@ -464,9 +478,14 @@
 	{
 		gchar	homename[PATH_MAX] = "", syscfg[PATH_MAX], whine[2 * PATH_MAX];
 
+		if((hpath = getenv("HOME_ETC")) != NULL) {
+			g_snprintf(homename, sizeof homename, "%s/%s", hpath, RCNAME);
+		}
+		else {
 		if((hpath = getenv("HOME")) != NULL)
 			g_snprintf(homename, sizeof homename, "%s/%s", hpath, RCNAME);
-
+		}
+		
 		g_snprintf(syscfg, sizeof syscfg, PATH_CFG G_DIR_SEPARATOR_S "%s", RCNAME + 1);
 		g_snprintf(whine, sizeof whine, _("Couldn't find any configuration file; checked\n"
 				"both \"%s\" and \"%s\".\n"
