diff -urN gentoo-0.11.5/Makefile gentoo-0.11.5.new/Makefile
--- gentoo-0.11.5/Makefile	Thu Apr 22 20:24:18 1999
+++ gentoo-0.11.5.new/Makefile	Mon Apr 26 13:50:47 1999
@@ -12,13 +12,13 @@
 #
 
 # Where to put binary on 'make install'?
-BIN	= /usr/local/bin
+BIN	= /usr/X11R6/bin
 
 # Where to put icons on 'make install'? Note: if you change this,
 # gentoo will not find the icons as it starts up. You're going to
 # have to alter gentoo's icon path (in the config window, "Paths"
 # tab) before it will work.
-ICONS	= /usr/local/lib/gentoo/
+ICONS	= /usr/X11R6/share/gentoo/
 
 #----------------------------------------- Hardcoding paths
 
@@ -27,13 +27,13 @@
 # the defined symbols proper C strings, of course.
 
 # Where should gentoo look for the system-global config file?
-PTH_CFG	= '"/usr/local/etc/"'
+PTH_CFG	= '"/usr/X11R6/share/gentoo/"'
 
 # Where should gentoo look for icons, if no path is configured?
 # This is typically just "icons", to look in the directory that
 # gentoo is started from, followed by some system-wide location
 # (which in my world is /usr/local/lib/gentoo/icons).
-PTH_ICN = '"icons:/usr/local/lib/gentoo/icons"'
+PTH_ICN = '"icons:/usr/X11R6/share/gentoo/icons"'
 
 # This path tells gentoo where to look for its GTK+ RC file.
 # The name of the file is always ".gentoogtkrc" (with dot), but
